build.gradle.kts

```diff
@@ -131,6 +142,36 @@ tasks {
 		group = MAIN
 	}
 	
+	val buildInstaller by creating(Install4jTask::class) {
+		dependsOn(shadowJar)
+		group = MAIN
+		
+		var path : File? = null
+		doFirst {
+			println("[INSTALL4J] Installers will be saved in ./install4j directory")
+			path = file(jarFile).copyTo(file("install4j/$jarFile"), true)
+			println(
```

Comment:
Why is this logged? In Gradle the standard is to not log anything if there is no problem. Otherwise use the gradle logger with the appropriate loglevel.

Reply:
Will remove.

Reply:
ef8587656e17d6e7fb927f12874a1b01153d9f78